xml_file_generation/generate_xml_drof.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# Copyright 2023 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Contact: Ezhilsabareesh Kannadasan
```

Comment:
Could you add a sentence describing what this script is for?

Reply:
Thanks Dougie, added in this commit [05a1fb7](https://github.com/COSIMA/om3-scripts/commit/05a1fb70bf4f00cb668bcc5cb3ff25f00c71261e)